app/views/orgs/roster_entries/assignment_repos/_linked_accepted.html.erb

```diff
@@ -10,7 +10,11 @@
     <div class='d-flex flex-column flex-justify-around'>
       <div>
         <%= link_to view.github_user_url, class: "assignment-repo-github-url", target: '_blank' do %>
-          <h3 class="css-truncate css-truncate-target"><%= view.github_user_login %></h3>
+          <% if defined? current_roster_entry %>
```

Comment:
Why are you using `if defined?` instead of `blank?` or `present?`? Does the controller not define it sometimes?

Reply:
Opps I didn't see this. I replied to a similar question. The controller does not always define it.